chain/epoch-manager/src/reward_calculator.rs

```diff
@@ -88,23 +88,88 @@ impl RewardCalculator {
         let mut epoch_actual_reward = epoch_protocol_treasury;
         let total_stake: Balance = validator_stake.values().sum();
         for (account_id, stats) in validator_block_chunk_stats {
-            // Uptime is an average of block produced / expected and chunk produced / expected.
+            // Uptime is an average of block produced / expected, chunk produced / expected,
+            // and chunk endorsed produced / expected.
+
+            let expected_blocks_produced = stats.block_stats.expected;
+            let expected_chunks_produced = stats.chunk_stats.expected();
```

Comment:
For readability, can we rename these to `expected_blocks` and `expected_chunks`? Because in formulas it makes me always wonder whether it is "expected" or "produced".

Reply:
Done in https://github.com/near/nearcore/pull/11121/commits/da37c4a2ebed08a66f563886bc7cb40470d06f9a